packages/renderer/src/lib/kube/KubePodDetailsSummary.svelte

```diff
@@ -1,21 +1,34 @@
 <script lang="ts">
 import type { V1Pod } from '@kubernetes/client-node';
 
+import Cell from '/@/lib/details/DetailsCell.svelte';
 import Table from '/@/lib/details/DetailsTable.svelte';
+import Title from '/@/lib/details/DetailsTitle.svelte';
 
+import type { EventUI } from '../events/EventUI';
+import EventsTable from './details/EventsTable.svelte';
 import KubeObjectMetaArtifact from './details/KubeObjectMetaArtifact.svelte';
 import KubePodSpecArtifact from './details/KubePodSpecArtifact.svelte';
 import KubePodStatusArtifact from './details/KubePodStatusArtifact.svelte';
 
 export let pod: V1Pod | undefined;
+export let events: EventUI[];
 </script>
 
 <Table>
   {#if pod}
     <KubeObjectMetaArtifact artifact={pod.metadata} />
     <KubePodStatusArtifact artifact={pod.status} />
     <KubePodSpecArtifact artifact={pod.spec} />
-  {:else}
+    <tr>
+      <Title>Events</Title>
+    </tr>
+    {#if events?.length}
+      <EventsTable events={events} />
+    {:else}
+      <tr><Cell>No events</Cell></tr>
+    {/if}
```

Comment:
Can we make a re-usage component, such as `KubeEventArtifact`, that we can share between Pods, Deployments, services etc.. (all kube object) ?

Reply:
added a reusable KubeEventArtifact component